content/blog/branch-patterns-automate-prs.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Branch patterns and automating PRs
+date: '2022-04-14T07:00:00.000Z'
+author: James Perkins
+prev: content/blog/read-only-tokens-content-anytime.md
+---
+
+When working on a git backed site you don’t normally want to work directly on your main branch. Ideally, you want to be able to finish your post or site update, check if it looks good, and schedule the release. A good branch pattern and scheduler is key to making this work. In this post, we are going to cover two different options to automate your PRs and an example of the branch pattern I use with Tina.
+
+## Branch Pattern
```

Comment:
What's the reasoning behind this section being mixed in the same post with the content scheduling stuff? Right now it kind of feels like two separate posts to me. If we split it out, I think "Scheduling content with Tina" sounds like a very interesting post on its own

Reply:
We can if we feel we could create a whole post that is worth branch patterns. Personally I felt that it was important, to understand why someone may need to schedule a PR.

Reply:
> Personally I felt that it was important, to understand why someone may need to schedule a PR.

I just kind of see the naming / use-of-staging-branch a bit out of scope. For instance, this post's PR could be named anything (and we don't use a staging branch) and you will still be able to schedule the PR to be merged using the approaches outlined.

Reply:
Sure. That is true. This would be useful when asking for input in the marketing channel for future inputs versus when we are getting ready to push posts.

Reply:
my 2 cents: I think if you just drop the Temporary Branches section it fits pretty well with the rest of the post

Reply:
and also maybe title it: Branching Workflow Automation or something like that

Reply:
Oh. sorry. I missed that. Didn't realize you were gearing up to push, just noticed the GitHub notifications. Disregard my comments!